### PR TITLE
Mibs/feature storybook

### PR DESCRIPTION
**Pull request by mibsbalsante**
- Add storybook to project and its config file
- Install a eslint and prettier dependency

### CODE REVIEW

**Review by dleitee — DISMISSED** [review]

**Comment by dleitee** [comment]
@mibsbalsante Approved, but I think we won't merge it right now, ok?

**Comment by mibsbalsante** [comment]
Ok 😄

**Comment by mibsbalsante** [comment]
Now is possible to use css modules with storybook 🎉🎉🎉
Please, let me know if there's something missing. 

@dleitee

**Comment by douglasgimli** [comment]
Awesome initiative and work @mibsbalsante 👏 👏 👏 👏

**Comment by douglasgimli** [comment]
Awesome initiative and work @mibsbalsante 👏 👏 👏 👏

**Comment by dleitee** [comment]
awesome!! I think that's it @mibsbalsante

**Review by dleitee — DISMISSED** [review]

File: postcss.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const autoprefixer = require('autoprefixer')
 const postCSSImport = require('postcss-import')()
 const postCSSNested = require('postcss-nested')
-const postCssCssVariables = require('postcss-css-variables')()
+const postCssCssVariables = require('postcss-css-variables')
```
Comment (douglasgimli) [inline postcss.config.js]: Small typo here, just to keep the pattern it's good to use `postCSS` instead of `postCss`

File: postcss.config.js [hunk 2]
```diff
@@ -13,14 +13,18 @@ const postCSSCustomMedia = require('postcss-custom-media')({
 const postCSSInlineSVG = require('postcss-inline-svg')()
 const colorFunction = require('postcss-color-function')
 
+const cssVariables = require('./src/config/css-variables')
```
Comment (douglasgimli) [inline postcss.config.js]: In my opinion, we can try to always use more declarative variable names, like in this case: `defaultCSSVariables`, does it make sense to you?

File: src/pages/_document.js [hunk 3]
```diff
@@ -6,6 +6,15 @@ export default class DefaultDocument extends Document {
     return (
       <html lang="en">
         <Head>
+          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+          <link
```
Comment (douglasgimli) [inline src/pages/_document.js:10]: Google Web Fonts has a library that we can programmatically include their web fonts, that way we don't need to hard code their fonts inside the header.

**Review by douglasgimli — DISMISSED** [review]
Small inline comments that we can try to improve in the future. But again, awesome initiative 👏 👏

File: postcss.config.js (hunk 2 above)
Comment (mibsbalsante) [reply postcss.config.js]: For sure! I've changed this now 😄

File: src/pages/_document.js (hunk 3 above)
Comment (dleitee) [reply src/pages/_document.js:10]: @douglasgimli which library is that?

**Review by dleitee — APPROVED** [review]

**Pull request merged**